### PR TITLE
Fixes #1401 - twitter and github icon

### LINKED ISSUE #1401: Replace Twitter text link with icon + add GH icon
We'd like to change the twitter link into the twitter logo icon and add a github logo icon link to the footer.

You can find a more detailed explanation here as well as the discussion: https://github.com/webcompat/webcompat.com/issues/1282#issue-200693908.

Template: https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/shared/footer.html

- [ ] add twitter icon SVG to footer http://www.flaticon.com/free-icon/twitter-logo-silhouette_8800
- [ ] link twitter icon to https://twitter.com/webcompat/with_replies
- [ ] add github icon SVG to footer https://octicons.github.com/icon/octoface/
- [ ] link github icon to https://github.com/webcompat/webcompat.com/

If you have any questions, please let us know! <3

### PR DESCRIPTION
**Pull request by nj4710**
See #1401 for more context.

* Added twitter SVG to footer.html.
* Linked twitter icon.
* Added github SVG to footer.html.
* Linked github icon.

Signed-off-by: Neha Jha <nj4710@gmail.com>

### CODE REVIEW

**Comment by magsout** [comment]
Hum, thanks @nj4710 I'm not the reviewer but I think svg inline is better.

**Comment by zoepage** [comment]
@magsout we decided on something different.

File: webcompat/templates/shared/footer.html [hunk 1]
```diff
@@ -26,7 +26,12 @@
       <ul class="wc-Footer-list r-ResetList">
         <li class="wc-Footer-item wc-Footer-item--follow">
           <a class="wc-Footer-link" href="https://twitter.com/webcompat/with_replies" target="_blank">
-            <span class="wc-Footer-link-label">Follow @WebCompat</span>
+            <span class="wc-Footer-link-label wc-Twitter"></span>
+          </a>
+        </li>
+        <li class="wc-Footer-item wc-Footer-item--follow">
+          <a class="wc-Footer-link" href=" https://github.com/webcompat/webcompat.com/" target="_blank">
+            <span class="wc-Footer-link-label wc-Github"></span>
```
Comment (zoepage) [inline webcompat/templates/shared/footer.html]: Empty spans are semantically not correct. Please remove.

File: webcompat/templates/shared/footer.html [hunk 2]
```diff
@@ -26,7 +26,12 @@
       <ul class="wc-Footer-list r-ResetList">
         <li class="wc-Footer-item wc-Footer-item--follow">
           <a class="wc-Footer-link" href="https://twitter.com/webcompat/with_replies" target="_blank">
-            <span class="wc-Footer-link-label">Follow @WebCompat</span>
+            <span class="wc-Footer-link-label wc-Twitter"></span>
```
Comment (zoepage) [inline webcompat/templates/shared/footer.html]: Empty spans are semantically not correct. Please remove.

**Comment by zoepage** [comment]
@nj4710 could you please change the commit title to "Issue #1401 - added twitter and github icon"?
You can change it with `git commit --amend`. Thanks!

**Comment by nj4710** [comment]
@zoepage  Can I substitute the span tag with a p tag? Is that correct? Do I need to change something else too? Thanks!

**Comment by zoepage** [comment]
@nj4710 I should have been more clear, sorry. (about to leave the office right now).
As an empty paragraph serves no real semantic purpose except of showing off a design element through css it wouldn't make sense either.

How about you leave the span and add "follow on twitter" and "see on github" to it, so we have content, but hide the text? Would this be ok?

Thanks && sorry again for the bad comment before.

**Comment by nj4710** [comment]
@zoepage Yes that would be amazing too. Thank you so much for your inputs.

File: webcompat/templates/shared/footer.html (hunk 1 above)
Comment (magsout) [inline webcompat/templates/shared/footer.html]: or you can add an `aria-hidden="true"`

**Comment by nj4710** [comment]
@zoepage Please do tell me if anything should be changed. Thanks :)

**Comment by zoepage** [comment]
@nj4710 On device with of e.g. 500px the twitter icon moves to the left as seen in the following:
screen shot 2017-03-09 at 3 28 03 pm

Caused by https://github.com/nj4710/webcompat.com/blob/issue1401/webcompat/static/css/development/components/footer.css#L80

To not make everything too complicated, just remove the `margin-left/right` at both and the `height`. Please add `background-repeat: no-repeat;` . Then we can merge and go from there. :)

**Comment by nj4710** [comment]
@zoepage I have removed `margin-left/right,` `height` and added `background-repeat: no-repeat`. Are we good?

**Comment by zoepage** [comment]
@nj4710 You did a really great job! Thank you for being so patient! :)

**Comment by karlcow** [comment]
@magsout @zoepage Looking at the comments:

> > Hum, thanks @nj4710 I'm not the reviewer but I think svg inline is better.
> @magsout we decided on something different.

1. So I was intrigued. I have read this issue. I don't see such decision.
2. Then hoping to https://github.com/webcompat/webcompat.com/issues/1401 to find the decision. (btw thanks @zoepage for caring about the paper trail. This is useful). No decision there.
3. But there's a link to https://github.com/webcompat/webcompat.com/issues/1282#issue-200693908 for context. Reading the full issue. There was a discussion about the icons in https://github.com/webcompat/webcompat.com/issues/1282#issuecomment-273308830

but still not decisions. If I understand there are a couple of ways of doing it.

* external SVG
* inline SVG
* icon-font

Did I miss anything? Maybe indeed we can [avoid icon fonts](https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/). Now in between inline SVG and external SVG I don't have strong preferences. once it is cached, it doesn't have impact on perf if external. inline SVG is well compressed.

**Comment by karlcow** [comment]
@nj4710 wonderful work.

**Comment by nj4710** [comment]
@karlcow Thank you so much!

**Pull request merged**